_ap/14/index.adoc

```diff
@@ -0,0 +1,14 @@
+---
+num: 14
+category: "SOPs"
+title: "Mitigating exposing sensitive content in upstream SOP repos "
+status: "Draft"
+authors:
+  - "Rachel Lawton"
+---
+
+When creating a upstream SOPs repo, care has to be taken to mitigate exposing sensitive content. A model that can be easily replicated to help with this mitigation is having a public upstream repo along with a private downstream repo. The public upstream repo will contain only information that is suitable for outside consumption, while the downstream private repo will have the content from the upstream repo as well as having extra information accompanying it that would not suitable for outside consumption. 
```

Comment:
Minor but if you could break line at the end of each sentence, it makes for a more diff readable / editable content.

Reply:
I would change "mitigate exposing sensitive content" to focus more on "enabling suitable content per context". See my previous comment for a more detailed explanation for this opinion.